Loggie/Classes/LoggieManager.swift

```diff
@@ -75,6 +76,26 @@ public class LoggieManager: NSObject, LogsDataSourceDelegate {
         return navigationController
     }
 
+    public func showLogs() -> UINavigationController {
```

Comment:
I think this one doesn't actually show it, it just returns what needs to be shown?

Can we call it `logsViewController` maybe?

Reply:
Also, can we reuse this one from:

`
public func showLogs(from viewController: UIViewController, filter: ((Log) -> Bool)? = nil) -> UINavigationController {
`